manager/src/main/java/com/redhat/service/bridge/manager/BridgesServiceImpl.java

```diff
@@ -121,24 +119,22 @@ public Bridge getBridge(String id, String customerId) {
 
     @Override
     public void deleteBridge(String id, String customerId) {
```

Comment:
Same comment as the need to maintain to `doCreateBridge` method.

Reply:
@robpblake Updates completed.